Class Presentations/reveal-presentations/SelfConf2016.html

```diff
@@ -0,0 +1,582 @@
+
+		<meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=no, minimal-ui">
+
+		<link rel="stylesheet" href="css/reveal.css">
+		<link rel="stylesheet" href="css/theme/black.css" id="theme">
+
+		<!-- Code syntax highlighting -->
+		<link rel="stylesheet" href="lib/css/zenburn.css">
+
+		<!-- Printing and PDF exports -->
+		<script>
+			var link = document.createElement( 'link' );
+			link.rel = 'stylesheet';
+			link.type = 'text/css';
+			link.href = window.location.search.match( /print-pdf/gi ) ? 'css/print/pdf.css' : 'css/print/paper.css';
+			document.getElementsByTagName( 'head' )[0].appendChild( link );
+		</script>
+
+		<!--[if lt IE 9]>
+		<script src="lib/js/html5shiv.js"></script>
+		<![endif]-->
+	</head>
+
+	<body>
+
+		<div class="reveal">
+
+			<!-- Any section element inside of this container is displayed as a slide -->
+			<div class="slides">
+				<section>
+					<h1>What Teaching Has Taught Us</h1>
+					<h3>Brandy Foster and Kyle Ofori</h3>
+				</section>
+
+				<section>
+					<h2>Welcome!</h2>
+				</section>
+
+				<section>
+					<h2>Goals</h2>
+					<ul class="checklist">
+						<li>
+							<input type="checkbox" id="goal0"  />
+							<label for="goal0">Who Are We, and What Is This Project?</label>
+						</li>
+						<li>
+							<input type="checkbox" id="goal1" />
+							<label for="goal1">Why Did We Decide to Do This?</label>
+						</li>
+						<li>
+							<input type="checkbox" id="goal2" />
+							<label for="goal2">What Resources Did We Use?</label>
+						</li>
+						<li>
+							<input type="checkbox" id="goal3" />
+							<label for="goal3">What Has Been Easy? Difficult? Fun?</label>
+						</li>
+						<li>
+							<input type="checkbox" id="goal4" />
+							<label for="goal4">What Are Some Things We Didn't Think About?</label>
+						</li>
+						<li>
+							<input type="checkbox" id="goal5" />
+							<label for="goal5">What Lessons Did We Learn?</label>
+						</li>
+					</ul>
+				</section>
+
+				<section>
+					<h2>Who Are We?</h2>
+				</section>
+
+				<section>
+					<h2>Brandy.</h2>
+					<img src="../presentation-images/selfconf2016/brandy.png" width="292px" height="405px">
+				</section>
+
+				<section>
+					<h2>Kyle.</h2>
+					<img src="../presentation-images/selfconf2016/kyle.jpg" width="292px" height="390px">
+				</section>
+
+				<section>
+					<h2>What Is This Project?</h2>
+				</section>
+
+				<section>
+				<p><strong>D-Code</strong> is an after-school program in which students
+					learn the fundamentals of coding &amp; software development.</p>
+				</section>
+
+				<section>
+				<p>The program is held at Detroit Edison Public School Academy (DEPSA)
+					Early College of Excellence, a charter high school just north of
+					Eastern Market.</p>
+					<img src="../presentation-images/selfconf2016/ece_triptych.png">
+				</section>
+
+				<section>
+				<p>We meet twice a week.</p>
```

Comment:
I don't want to give out so much detail, so I might just write "for two hours each after school." Maybe I'm paranoid, but something about pictures + address + exactly where and when to find us seems like too much information...I wonder if we should not specify the name of the school, either.